lib/active_pdftk/call.rb

```diff
@@ -196,7 +196,7 @@ def pdftk(dsl_statements = {})
         # We ignore 'no info dictionary' warning since it doesn't affect the integrity of the PDF
         # and handling this warning as an error prevents us from accessing the other metadata
         raise(CommandError, {:stderr => @error, :cmd => cmd, :stdout => stdout, :output => @output}) unless
-          ((@error = stderr.read).empty? || @error == "Warning: no info dictionary found\n")
+          ((@error = stderr.read).empty? || @error.include?("Warning: no info dictionary found"))
```

Comment:
hey @trgeiger I do wonder if it will be good in do `.include?("Warning")` so all warning dont kill our PDF generation.